doc/guides/dts/howtos.rst

```diff
@@ -114,11 +113,11 @@ this:
    #if DT_HAS_NODE_STATUS_OKAY(MY_SERIAL)
    struct device *uart_dev = device_get_binding(DT_LABEL(MY_SERIAL));
    #else
-   #error "Node is disabled or has no matching binding"
+   #error "Node is disabled, has no matching binding, or initialization failed"
    #endif
 
-If you see the ``#error`` output, something is wrong with either your
-devicetree or bindings.
+If you see the ``#error`` output, either something is wrong with either your
```

Comment:
remove second either in the line

Reply:
No, this is correct.  There are two optional subordinate clauses, one of which requires its own `either`.

Reply:
Or rather, it's correct with two eithers, just the first, just the second, or neither.  IMO.  English is pretty flexible.